### DISCUSSION TITLE
Can the locking for duplicated jobs work in several Sidekiq instances?

### DISCUSSION THREAD

**Discussion by huyvohcmc** [mhenrixon/sidekiq-unique-jobs#747] [Q&A] [answered]
Basically the question, I'm wondering if Job1 and Job2 are the same but belong to different replicas, can we guarantee only one job is executed at a time, and the other job will wait until the current job is finished?

**Comment by mhenrixon** [comment, marked as answer]
Yup 👍 that's exactly how it is supposed to work. The only requirement is 1 Redis instance. The amount of sidekiq processes should be irrelevant.
